frontend/src/pages/meditation/components/meditation-player/meditation-player.tsx

```diff
@@ -1,41 +1,61 @@
-import { AudioPlayer } from '#libs/components/components.js';
-import { useCallback, useState } from '#libs/hooks/hooks.js';
+import { AudioPlayer, Loader } from '#libs/components/components.js';
+import { DataStatus } from '#libs/enums/data-status.enum.js';
```

Comment:
```suggestion
import { DataStatus } from '#libs/enums/enums.js';
```

Reply:
Done, you remind me that I don't need data status there